### PR TITLE
https://issues.redhat.com/browse/ACM-11824

### PR DESCRIPTION
**Pull request by oafischer**
2.11 and 2.10

### CODE REVIEW

**Review by birsanv — APPROVED** [review]
lgtm

File: business_continuity/backup_restore/backup_arch.adoc [hunk 1]
```diff
@@ -37,7 +37,7 @@ kind: MyResource
     cluster.open-cluster-management.io/backup: ""
 ----
 +
-*Note:* Secrets used by the `hive.openshift.io.ClusterDeployment` resource need to be backed up, and are automatically annotated with the `cluster.open-cluster-management.io/backup` label only when the cluster is created using the console. If the Hive cluster is deployed using GitOps instead, the `cluster.open-cluster-management.io/backup` label must be manually added to the secrets used by the `ClusterDeployment`.
+*Note:* Secrets used by the `hive.openshift.io.ClusterDeployment` resource need to be backed up, and are automatically annotated with the `cluster.open-cluster-management.io/backup` label only when you create the cluster by using the console. If you deploy the Hive cluster by using GitOps instead, you must manually add the `cluster.open-cluster-management.io/backup` label to the secrets used by the `ClusterDeployment`. Secret and ConfigMap resources with the `cluster.open-cluster-management.io/backup: cluster-activation` label are restored at cluster activation time.
```
Comment (dockerymick) [inline business_continuity/backup_restore/backup_arch.adoc]:
```suggestion
*Note:* Secrets used by the `hive.openshift.io.ClusterDeployment` resource need to be backed up, and are automatically annotated with the `cluster.open-cluster-management.io/backup` label only when you create the cluster by using the console. If you deploy the Hive cluster by using GitOps instead, you must manually add the `cluster.open-cluster-management.io/backup` label to the secrets used by the `ClusterDeployment` resource. Secret and config map resources with the `cluster.open-cluster-management.io/backup: cluster-activation` label are restored at cluster activation time.
```

**Review by dockerymick — APPROVED** [review]
left a suggestion, thanks!

**Review by jc-berger — APPROVED** [review]
/lgtm

after applying Mikela's suggestions.

**Pull request merged**